assistedInstallerService.py

```diff
@@ -183,6 +183,20 @@ def wait_for_api(self) -> None:
             count += 1
             time.sleep(2)
 
+    def stop(self) -> None:
+        lh = host.LocalHost()
+        result = lh.run("podman pod ps --format json")
+        if result.err:
+            print("Error {result.err}")
+            exit(1)
+        name = "assisted-installer"
+        if name in map(lambda x: x["Name"], json.loads(result.out)):
+            print(f"{name} already running, stopping it before restarting")
+            lh.run(f"podman pod stop {name}")
+            lh.run(f"podman pod rm {name}")
+        else:
+            print(f"{name} not yet running")
+
```

Comment:
Don't want this since you might have multiple clusters on the same node. Can you elaborate why we would need to tear down assisted installer?

Reply:
I think it is good to be able to really clean the systems i.e. removing everything we installed.
However, I got your point. So, we might just need an extra option to delete the ai - so it's not deleted automatically, but we could do it if we want to.

Reply:
Let's add `--teardown-full` that also tears down assisted installer.